src/posix.mak

```diff
@@ -90,11 +90,12 @@ else
   # Auto-bootstrapping, will download dmd automatically
   # Keep var below in sync with other occurrences of that variable, e.g. in circleci.sh
   HOST_DMD_VER=2.072.2
+  HOST_DMD_YEAR=2016
   HOST_DMD_ROOT=$(TMP)/.host_dmd-$(HOST_DMD_VER)
   # dmd.2.072.2.osx.zip or dmd.2.072.2.linux.tar.xz
   HOST_DMD_BASENAME=dmd.$(HOST_DMD_VER).$(OS)$(if $(filter $(OS),freebsd),-$(MODEL),)
   # http://downloads.dlang.org/releases/2.x/2.072.2/dmd.2.072.2.linux.tar.xz
-  HOST_DMD_URL=http://downloads.dlang.org/releases/2.x/$(HOST_DMD_VER)/$(HOST_DMD_BASENAME)
+  HOST_DMD_URL=https://s3-us-west-2.amazonaws.com/downloads.dlang.org/releases/$(HOST_DMD_YEAR)/$(HOST_DMD_BASENAME)
```

Comment:
> The website would just do a redirect. It would be no more of a single point of failure than the dlang.org website.

I think the best solution would be to either
- upload releases to GH releases too
- restructure the download,s.t. they are year independent
- use the releases to a provider that supports SSL out-of-the-box

Note that we could use Cloudfront on AWS to achieve SSL, but sadly that [would/might be really expensive](https://aws.amazon.com/cloudfront/pricing/) (not sure how traffic is currently generated).

> because apparently Jan has no problem setting SSL up.

https://dconf.org/